src/widgets/hits/hits.js

```diff
@@ -41,7 +44,11 @@ function hits({
     hitsPerPage = 20
   } = {}) {
   if (!container) {
-    throw new Error(usage);
+    throw new Error('Must provide a container.' + usage);
+  }
+
+  if (templates.item && templates.items) {
+    throw new Error('Must contain only items OR item template.' + usage);
```

Comment:
Good question. I first tried items, but it was too visually similar and I though it would be prone to errors. I didn't tried full, but it's an adjective.